takwimu/static/css/takwimu.scss

```diff
@@ -145,10 +146,27 @@ a {
 }
 
 
+/*
+* Custom CSS
+* Topics tabs css
+*/
+.custom-nav a {
+  border-bottom: 0.4em solid transparent;
+  cursor: pointer;
+  text-decoration: none;
+  &:hover, &.active {
+    background: $brand-extend-primary;
+    border-bottom: 0.4em solid $brand-extend-white;
```

Comment:
@mtuchi sticking with the current approach, we can have this in an in-template `style` tag.